packages/codemods/src/helpers/index.ts

```diff
@@ -1,4 +1,5 @@
 import { errorMessage, infoMessage, logMessage } from './message';
 import { _dirname } from './path';
+import { removeCircleBrackets } from './removeCircleBrackets';
 
-export { _dirname, errorMessage, infoMessage, logMessage };
+export { _dirname, errorMessage, infoMessage, logMessage, removeCircleBrackets };
```

Comment:
**suggestion (blocking):** Please use correct naming.

Types of Brackets:
- Parentheses ( )
- Square brackets [ ]
- Curly brackets { }
- Angle brackets ⟨ ⟩

My suggestion is to use `removeParentheses`.